components/DynamicForm/ChooseFormType.tsx

```diff
@@ -4,11 +4,13 @@ import { Question, Answer, AnswersKeys, answers } from '../../models'
 import { MyRadio } from './MyRadio'
 import { MyResult } from './MyResult'
 import { FormAnswer } from '../../utils/FormContext'
+import MyContinue from './MyContinue'
```

Comment:
Notes for people reviewing: Ignore this file, these are changes from the other PR